### ISSUE TITLE
Build system errors

### ISSUE THREAD

**Issue by 6r1d** [paroj/gltut#125]
Hello. I have tried to build the whole tutorial under Linux, unsuccessfully, after generating the makefiles using `premake4 gmake`. The library names seem to be incorrect:

```
framework.cpp
Timer.cpp
Linking framework
==== Building Tut 01 Main (debug) ====
Linking Tut 01 Main
/usr/bin/ld: cannot find -lglutilD
/usr/bin/ld: cannot find -lfreeglutD
/usr/bin/ld: cannot find -lglmeshD
/usr/bin/ld: cannot find -lglimgD
/usr/bin/ld: cannot find -lglloadD
collect2: error: ld returned 1 exit status
Makefile:85: recipe for target 'Tut 01 MainD' failed
make[1]: *** [Tut 01 MainD] Error 1
Makefile:16: recipe for target 'Tut 01 Main' failed
make: *** [Tut 01 Main] Error 2
```

**Comment by paroj** [comment]
fixed with db01d1c5d255a9e819facb9dcfc47cd560b27268

**Comment by 6r1d** [comment]
Thanks :-)
